LayoutTests/inspector/model/source-map-spec.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../http/tests/inspector/resources/inspector-test.js"></script>
+<script>
+const baseSourceMapTestDir = "../../imported/tc39-tg4/source-map-tests"
+function triggerScriptResource(scriptName) {
+    let script = document.createElement("script");
+    script.src = `${baseSourceMapTestDir}/resources/${scriptName}`;
+    document.body.appendChild(script);
+}
+
```

Comment:
In the latest version of the patch, I used some AJAX here to fetch the test data. The test cases are generated and run in a callback for the GET, so it works fine without needing async functions or promises. This lets each source map test be its own web inspector TestSuite test case.